integration/compatibility/data_sets.py

```
@@ -10,6 +10,7 @@
 
 SETS_KEY = lambda key_type: f"{key_type} sets"
 CREATES_KEY = lambda key_type: f"{key_type} creates"
+SETUP_KEY = lambda key_type: f"{key_type} setup"
```

Ruff (0.15.13)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `SETUP_KEY` as a `def`

(E731)